velox/common/file/FileStream.cpp

```diff
@@ -14,18 +14,129 @@
  * limitations under the License.
  */
 
-#include "velox/common/file/FileInputStream.h"
+#include "velox/common/file/FileStream.h"
 
 namespace facebook::velox::common {
 
+namespace {
+#ifdef __APPLE__
+// Due to macOS bug, we need to set read/write max.
```

Comment:
Arrow indicates there is a bug in MacOS, https://github.com/apache/arrow/pull/1909/files#diff-2bee38c6d7191943438be259488a590b77fb431978b5e72049dd668735c73385R78, but I don't know what's the bug.